features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/RumMonitor.kt

```diff
@@ -323,6 +323,15 @@ interface RumMonitor {
      */
     fun stopSession()
 
+    /**
+     * Adds view loading time to current RUM view based on the time elapsed since the view was started.
+     * This method should be called only once per view.
+     * If the view is not started, this method does nothing.
+     * If the view is not active(stopped), this method does nothing.
```

Comment:
> [!NOTE]
> Since the loading time is applied to the `current RUM view` (maybe should be the `current active RUM View`), those two lines are misleading IMO. It feels like we make the assumption that a view is necessarily an Activity/Fragment with a lifecycle (start, resume, …), which is not always the case.

```suggestion
     * If no view is started or active, this method does nothing.
```

Reply:
I am following the same docs as in iOS